### PR TITLE
IBX-5640: Added available translations to dashboard data

### PR DESCRIPTION
**Pull request by ViniTou**
| Question      | Answer
| ------------- | ---
| Tickets       | https://issues.ibexa.co/browse/IBX-5640
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes/no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/lib/Tab/Dashboard/PagerLocationToDataMapper.php [hunk 1]
```diff
@@ -84,4 +85,22 @@ private function getVersionContributor(VersionInfo $versionInfo): ?User
...
+    ): iterable {
+        $availableTranslationsLanguages = $this->languageService->loadLanguageListByCode(
+            $versionInfo->languageCodes
+        );
+
+        return array_filter(
+            $availableTranslationsLanguages,
+            (static function (Language $language): bool {
+                return $language->enabled;
+            })
```
Comment (konradoboza) [inline src/lib/Tab/Dashboard/PagerLocationToDataMapper.php]:
```suggestion
            static function (Language $language): bool {
                return $language->enabled;
            }
```

File: src/lib/Tab/Dashboard/PagerLocationToDataMapper.php [hunk 2]
```diff
@@ -84,4 +85,22 @@ private function getVersionContributor(VersionInfo $versionInfo): ?User
             return null;
         }
     }
+
+    /**
+     * @return iterable<\eZ\Publish\API\Repository\Values\Content\Language>
+     */
+    private function getAvailableTranslations(
+        VersionInfo $versionInfo
+    ): iterable {
```
Comment (konradoboza) [inline src/lib/Tab/Dashboard/PagerLocationToDataMapper.php]: Is there any specific reason why not:
```suggestion
    ): array {
```
?

Comment (ViniTou) [reply src/lib/Tab/Dashboard/PagerLocationToDataMapper.php]: No, I think I just adjusted some existing method, will fix.

**Review by konradoboza — APPROVED** [review]

**Review by Steveb-p — APPROVED** [review]

**Review by webhdx — APPROVED** [review]

**Review by jwibexa — APPROVED** [review]
Tested and validated on 3.3 and 4.4 experience

**Pull request merged**